springwolf-plugins/springwolf-kafka-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/KafkaMessageBindingProcessor.java

```diff
@@ -0,0 +1,62 @@
+package io.github.stavshamir.springwolf.asyncapi.scanners.channels.operationdata;
+
+import com.asyncapi.v2.binding.kafka.KafkaMessageBinding;
+import io.github.stavshamir.springwolf.asyncapi.scanners.channels.operationdata.annotation.KafkaAsyncOperationBinding;
+import io.github.stavshamir.springwolf.asyncapi.scanners.channels.operationdata.annotation.KafkaAsyncOperationBinding.KafkaAsyncMessageBinding;
+import io.github.stavshamir.springwolf.asyncapi.scanners.channels.operationdata.annotation.MessageBindingProcessor;
+import io.swagger.v3.oas.models.media.Schema;
+import io.swagger.v3.oas.models.media.StringSchema;
+import org.springframework.context.EmbeddedValueResolverAware;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.util.StringValueResolver;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Optional;
+
+@Component
+public class KafkaMessageBindingProcessor implements MessageBindingProcessor, EmbeddedValueResolverAware {
+    private StringValueResolver resolver;
+
+    @Override
+    public void setEmbeddedValueResolver(StringValueResolver resolver) {
+        this.resolver = resolver;
+    }
+
+    @Override
+    public Optional<ProcessedMessageBinding> process(Method method) {
+        return Arrays.stream(method.getAnnotations())
+                .filter(annotation -> annotation instanceof KafkaAsyncOperationBinding)
+                .map(annotation -> (KafkaAsyncOperationBinding) annotation)
+                .findAny()
+                .map(this::mapToMessageBinding);
+    }
+
+    private ProcessedMessageBinding mapToMessageBinding(KafkaAsyncOperationBinding bindingAnnotation) {
+        KafkaAsyncMessageBinding messageBinding = bindingAnnotation.messageBinding();
+        KafkaMessageBinding kafkaMessageBinding = KafkaMessageBinding.builder()
+                .bindingVersion(resolveOrNull(messageBinding.bindingVersion()))
```

Comment:
Should this fallback to operation binding?
They should be identical, right?

Reply:
Not sure what you mean by "fallback", do you mean that the actual value of the binding should be taken from the operation binding? If thats the case I'm not sure if it's intended that they need/should be the same. The Spec says 

> The version of this binding. If omitted, "latest" MUST be assumed. 

And the message binding is not part of the operation binding but an independent part of the operation.

Thats also one issue I have with the annotation: It makes it look like the message binding is part of the operation binding, although it is not. I also thought about adding an extra annotation for it, but I somehow feel that it would complicate things even more because you can't really discover what you can do by looking at one extra annotation and I see that a lot of annotations could stack up over time and make the code less readable.

Another option would be to have new annotation like AsyncKafka which then can contain the annotation for operation binding and message binding. What do you think?